src/main/java/com/mparticle/kits/BranchUtil.java

```diff
@@ -129,7 +137,7 @@ Double readOutDouble(String key) {
     // Region Translate MPEvents
 
     BranchEvent createBranchEventFromMPEvent(MPEvent mpEvent) {
-        BranchEvent branchEvent = createBranchEventFromEventName(mpEvent.getEventType().name());
+        BranchEvent branchEvent = createBranchEventFromEventName(mpEvent.getEventName());
```

Comment:
MPEvents will always semantically mean a custom event. They can only be created with a custom event name. For that reason, MPEvents always just generate a BranchEvent with the same name as the MPEvent.

CommerceEvents on the other hand have very specific types, and do not support custom naming schemes - so those are appropriate to map to Branch built-in names as you've done.